code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -0,0 +1,456 @@
+/datum/game_mode
```

Comment:
while it's a game-mode, in theory, in practice it's far more like an event----generally speaking, that's what players will know of it, too.

The event utilizes a lot of the game-modes procs for equipping/setting up things.